src/pages/workspace/accounting/qbo/export/QuickbooksNonReimbursableDefaultVendorSelectPage.tsx

```diff
@@ -73,7 +73,6 @@ function QuickbooksNonReimbursableDefaultVendorSelectPage({policy}: WithPolicyCo
             <ScreenWrapper testID={QuickbooksNonReimbursableDefaultVendorSelectPage.displayName}>
                 <HeaderWithBackButton title={translate('workspace.qbo.defaultVendor')} />
                 <SelectionList
-                    headerContent={<Text style={[styles.ph5, styles.pb5]}>{translate('workspace.qbo.defaultVendorDescription')}</Text>}
```

Comment:
The import line for `import Text` on line 9 needs to be removed in this file

Reply:
Ok I think I did that too!